earth_enterprise/src/support/geecheck/geecheck.pl

```diff
@@ -125,7 +125,8 @@ BEGIN
                       "5.1.3" => ["1", 0],
                       "5.2.0" => ["1", 0],
                       "5.2.1" => ["1", 0],
-                      "5.2.2" => ["1", 0],                      
+                      "5.2.2" => ["1", 0],
+		      "5.2.3" => ["1", 0],                      
```

Comment:
Should remove this comma.

Reply:
It looks like trailing commas is expected in Perl: [Perl Best Practices](http://my.safaribooksonline.com/book/programming/perl/0596001738/2dot-code-layout/commas?reader=pf&readerfullscreen=&readerleftmenu=1)

I'm fine with removing it but this is the way that version numbers were added in the past

Reply:
I'll second the Best Practices book on trailing commas in Perl.